scss/_navbar.scss

```diff
@@ -74,6 +74,13 @@
     position: static;
     float: none;
   }
+
+  .dropdown-toggle {
```

Comment:
For me this change is unnecessary. Why we need redefine `.dropdown-toggle` in .navbar?

Reply:
because without that we cannot have the up caret

Reply:
I just tested and works without these lines.
Still I don't understand why this is needed :)